### PR TITLE
vscode: fix symlink, add missing dependencies

### PR DESCRIPTION
**Pull request by shizonic**

### CODE REVIEW

File: srcpkgs/vscode/REMOVE [hunk 1]
```diff
@@ -7,6 +7,12 @@ if [ "$UPDATE" = "no" ]; then
             if [ -d usr/share/code-oss ]; then
                 rm -rf usr/share/code-oss
             fi
+            if [ -e usr/share/applications/code-oss.desktop ]; then
+                rm -f usr/share/applications/code-oss.desktop
+            fi
+            if [ -e usr/share/pixmaps/code-oss.png ]; then
+                rm -f usr/share/pixmaps/code-oss.png
+            fi
```
Comment (jnbr) [inline srcpkgs/vscode/REMOVE]: why do you remove them manually?

Comment (shizonic) [reply srcpkgs/vscode/REMOVE]: Couldn't it be that they still exist from an installation before switching from /usr/share to /usr/lib?

Comment (jnbr) [reply srcpkgs/vscode/REMOVE]: yes but xbps-install should remove them on update

Comment (shizonic) [reply srcpkgs/vscode/REMOVE]: Ahh okay. I didn't know that. Will fix it asap.

Comment (shizonic) [reply srcpkgs/vscode/REMOVE]: Done.

**Pull request merged**